package.json

```diff
@@ -60,6 +60,7 @@
     "webpack": "^4.29.6"
   },
   "dependencies": {
+    "lodash": "^4.17.11",
```

Comment:
This appears to not be used. Let's remove it.

Reply:
Done